geometry_msgs/Point point
float32 theta
float32 beta
float32 gamma
uint32 distance